inbox/encryption/vault.py

```diff
@@ -0,0 +1,76 @@
+import base64
+import os
+import time
+
+import hvac
+
+from inbox.config import config
+
+vault_config = config.get('VAULT')
+
+
+client = hvac.Client(url=vault_config['URL'])
+
+client.auth_approle(vault_config['APP_ROLE_ID'], vault_config['APP_ROLE_SECRET_ID'])
+
+
+def encrypt(plaintext, namespace_public_id):
+    start = time.time()
+
+    if not plaintext:
+        return ''
+
+    if not namespace_public_id:
+        return ''
+
+    # TODO: add proper handling of exception
+    try:
+        result = client.write(
+            # 'transit/encrypt/' + namespace_public_id,
+            'transit/encrypt/some-name-key',
+            plaintext=base64.b64encode(plaintext)
+        )
+
+        end = time.time()
+        latency_millis = (end - start) * 1000
+        print ''.join(['[vault] encryption latency is: ', str(latency_millis), 'ms'])
```

Comment:
May be we should add some "debug" key. Just to avoid the printing of that for each encrypted message?

Reply:
I'm going to replace it with metric to cloud watch. Will do that before merging to master.